src/renderer/lib/validate.ts

```diff
@@ -1,4 +1,4 @@
-export const isValidIpAddress = (ip: string) => {
+export const validateIpAddress = (ip: string): boolean => {
```

Comment:
```suggestion
export const isValidIpAddress = (ip: string): boolean => {
```

This returns only a boolean so it should still be called `isValidIpAddress`.